parse/string_set.go

```diff
@@ -1,10 +1,12 @@
-package parsestring
+package parse
 
 import (
 	"fmt"
 )
 
-func ParseStringSet(s string) (map[string]struct{}, error) {
+// StringSet splits a comma separated strings and constructs a
```

Comment:
This is awkwardly worded. `strings` is plural, but `splits a` indicates singular. I making it `string would fix it:
```suggestion
// StringSet splits a comma separated string and constructs a
```